deltachat-rpc-client/tests/test_iroh_webxdc.py

```diff
@@ -184,3 +184,26 @@ def thread_run():
         if event.kind == EventType.WEBXDC_REALTIME_DATA:
             assert int(bytes(event.data).decode()) > n
             break
+
+
+def test_no_reordering(acfactory, path_to_webxdc):
+    """Test that sending a lot of realtime messages does not result in reordering."""
+    ac1, ac2 = acfactory.get_online_accounts(2)
+    ac1.set_config("webxdc_realtime_enabled", "1")
+    ac2.set_config("webxdc_realtime_enabled", "1")
+
+    ac1_webxdc_msg, ac2_webxdc_msg = setup_realtime_webxdc(ac1, ac2, path_to_webxdc)
+
+    setup_thread_send_realtime_data(ac1_webxdc_msg, b"hello")
+    wait_receive_realtime_data([(ac2_webxdc_msg, b"hello")])
+
+    for i in range(200):
+        ac1_webxdc_msg.send_webxdc_realtime_data([i])
+
+    for i in range(200):
+        while 1:
+            event = ac2.wait_for_event()
+            if event.kind == EventType.WEBXDC_REALTIME_DATA and bytes(event.data) != b"hello":
```

Comment:
Shouldn't the event with "hello" be consumed by `wait_receive_realtime_data([(ac2_webxdc_msg, b"hello")])` above? And can it be emitted several times?